diff --git a/node_modules/tempo.ts/dist/wagmi/Connector.js b/node_modules/tempo.ts/dist/wagmi/Connector.js
index 52947bd..5db2620 100644
--- a/node_modules/tempo.ts/dist/wagmi/Connector.js
+++ b/node_modules/tempo.ts/dist/wagmi/Connector.js
@@ -156,7 +156,7 @@ export function webAuthn(options) {
             const credential = await config.storage?.getItem('webAuthn.activeCredential');
             if (!credential)
                 return;
-            account = Account.fromWebAuthnP256(credential);
+            account = Account.fromWebAuthnP256(credential, { rpId: options.rpId });
         },
         async connect(parameters = {}) {
             const capabilities = 'capabilities' in parameters ? (parameters.capabilities ?? {}) : {};
@@ -278,7 +278,7 @@ export function webAuthn(options) {
             })();
             config.storage?.setItem('webAuthn.lastActiveCredential', normalizeValue(credential));
             config.storage?.setItem('webAuthn.activeCredential', normalizeValue(credential));
-            account = Account.fromWebAuthnP256(credential);
+            account = Account.fromWebAuthnP256(credential, { rpId: options.rpId });
             if (keyPair) {
                 accessKey = Account.fromWebCryptoP256(keyPair, {
                     access: account,
